test/e2e/test-errortracker.js

```diff
@@ -104,303 +116,311 @@ describe('Error Tracker Server', () => {
     sandbox.restore();
   });
 
-  describe('rejects bad requests', () => {
-    it('without referrer', () => {
-      return makeRequest('', knownGoodQuery).then(() => {
-        throw new Error('UNREACHABLE');
-      }, (err) => {
-        expect(err.message).to.equal('Bad Request');
-      });
-    });
+  method("GET", makeRequest);
```

Comment:
Can we rename the `makeRequest()` function to `makeGetRequest()`? Avoids conflation with the parameter of the same name in `method()` below.

Reply:
Done.